<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Student</class>
 <widget class="QWidget" name="Student">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>40</y>
     <width>441</width>
     <height>581</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>470</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正粗圆宋简体</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>修改信息</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NamelineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正粗圆宋简体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>姓名</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正粗圆宋简体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>学号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正粗圆宋简体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>电话</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NumberlineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="TellineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="YuanxilineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>280</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>290</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正粗圆宋简体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>院系</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>330</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正粗圆宋简体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>专业</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ZhuanyelineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>330</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>510</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正粗圆宋简体</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>退出系统</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>390</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正粗圆宋简体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>寝室</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="QinshilineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>380</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ClasslineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>430</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>440</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正粗圆宋简体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>班级</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SexlineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正粗圆宋简体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>性别</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_user">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
